tests/static-checks.sh

```diff
@@ -500,13 +500,16 @@ check_url()
 	local invalid_urls_dir="${2:-}"
 	[ -n "$invalid_urls_dir" ] || die "need invalid URLs directory"
 
-	local curl_out=$(mktemp)
+	local curl_out
```

Comment:
@jodh-intel Is there a benefit for doing it this way..declaring and then defining later?

Reply:
Yep - see https://www.shellcheck.net/wiki/SC2155.